README.md

```diff
@@ -46,35 +46,36 @@ gradle release
 ```
 gradle test jacocoTestReport
 ```
-
-
-##Legacy Build (DEPRECATED!)
 ### Regenerate Javadocs
 
 ```
-rm -r <devcenter.home>/content/reference/android/api/*
-cd <project.home> 
-mvn -Pdev javadoc:javadoc install
+<devcenter.home> should be located in this relative path ../../../<devcenter.home> relatively <java-library.home>.
+cd <devcenter.home> 
+git pull devcenter
+cd <java-library.home>
+gradle release
 ```
 
 ### Release
 
 ```
-mvn -Prelease clean install
+cd <java-library.home>
+gradle release
 ```
 
 ###Explicit release steps (including the above)
 ```
-find and replace on version number (all poms.xml and RequestHeader version)
+find and replace on version (build.gradle in project directory and RequestHeader version)
```

Comment:
I see that in KinveyHeaders it's already changed to take the version from the gradle, so in comments should be specified only to change in build.gradle